NeuroID/src/main/java/com/neuroid/tracker/utils/NIDMetaData.kt

```diff
@@ -26,7 +35,7 @@ class NIDMetaData(context: Context) {
     private val isJailBreak: Boolean
     private var isWifiOn: Boolean?
     private val isSimulator: Boolean
-
+    private val gpsCoordinates: NIDLocation = NIDLocation(-1.0, -1.0, "none")
```

Comment:
should we make this status unknown since we aren't sure at first?

Reply:
yeah good suggestion will do that.